lib/Doctrine/DBAL/Schema/MySqlSchemaManager.php

```diff
@@ -176,25 +177,29 @@ protected function _getPortableTableColumnDefinition($tableColumn)
                 break;
         }
 
-        $length = ((int) $length == 0) ? null : (int) $length;
+        if ($this->_platform instanceof MariaDb102Platform) {
+            $columnDefault = $this->getMariaDb1027ColumnDefault($this->_platform, $tableColumn['default'] ?? null);
+        } else {
+            $columnDefault = (isset($tableColumn['default'])) ? $tableColumn['default'] : null;
```

Comment:
Use `??`.

Reply:
Previous comment from morozov... $tableColumn['default'] is always available. Fixed but in a different way:
``` php
if ($this->_platform instanceof MariaDb1027Platform) {
    $columnDefault = $this->getMariaDb1027ColumnDefault($this->_platform, $tableColumn['default']);
} else {
    $columnDefault = $tableColumn['default'];
}
```
Tried to follow previous reviews, don't overwritte local vars. In case the else looks superfluous